evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -802,8 +815,8 @@ def test_input_feature_names(example_graph):
         "column_1_c",
     ]
     assert input_feature_names["Logistic Regression"] == [
-        "Random Forest.x",
-        "Elastic Net.x",
+        "1_Random Forest.x",
```

Comment:
Do we perhaps have any other thoughts for options to name this?  The prepending with the underscore is kinda weird, especially with the space after it.  I think an appen....sion? might look better and adding like a "col" or something to it might be a bit clearer.

Reply:
I chose to do `column_name_component` scheme since to my knowledge we aren't using underscores anywhere else for the pipeline name. However, in general I don't really have any strong preferences for what the naming convention should be and I'm definitely open to ideas. Your idea of something like [`Col 0 Random Forest.x`, `Col 1 Random Forest.x`, etc.] makes sense to me and I'm good to move forward with this if there aren't any other suggestions.